### PR TITLE
Deployment portability improvements

### PR DESCRIPTION
**Pull request by tsibley**
See commit messages for details.

### CODE REVIEW

File: latex/style.tt [hunk 1]
```diff
@@ -1,5 +1,11 @@
 \usepackage{fontspec}
-\setmainfont{Helvetica Neue}
+\setmainfont{DroidSans}[
+  Extension   = .ttf,
+  UprightFont = *,
+  BoldFont    = *-Bold ]
+\setmonofont{DroidSansMono}[
```
Comment (silby) [inline latex/style.tt]: Is designating the mono font a forward-looking change for something else you want to add? The app has no mechanism for varying the typeface in a label.

**Comment by silby** [comment]
wow, I hate the UI of the github pull request conversation flow it turns out, why is there a huge box up there telling me continuous integration has not been set up???

File: latex/style.tt (hunk 1 above)
Comment (tsibley) [reply latex/style.tt]: Ahh, yes, it is. Good deduction! The code 128 barcoding I have working locally (and will push next week once I sort out some janky bits) uses `\tt` for text underneath the barcode. I can rework this PR and move the monospace font definition to the barcode bits if you'd like.

**Comment by silby** [comment]
Yeah, let’s port that forward to when it’s relevant then.

NB I also observed that the `droid` ctan package is included with texlive’s “full” installation but doing a full installation is not necessarily what one wants to do on the server. When I eventually write a README for this I will note that the package for droid can be installed with `tlmgr`, and maybe make the font configurable in a config file, but TKTKTK.

On Nov 9, 2018, at 4:34 PM, Thomas Sibley <notifications@github.com<mailto:notifications@github.com>> wrote:


@tsibley commented on this pull request.

________________________________

In latex/style.tt<https://github.com/MullinsLab/Lab-Labels/pull/1#discussion_r232430910>:

> @@ -1,5 +1,11 @@
 \usepackage{fontspec}
-\setmainfont{Helvetica Neue}
+\setmainfont{DroidSans}[
+  Extension   = .ttf,
+  UprightFont = *,
+  BoldFont    = *-Bold ]
+\setmonofont{DroidSansMono}[


Ahh, yes, it is. Good deduction! The code 128 barcoding I have working locally (and will push next week once I sort out some janky bits) uses \tt for text underneath the barcode. I can rework this PR and move the monospace font definition to the barcode bits if you'd like.

**Comment by tsibley** [comment]
Repushed the last commit.

**Comment by tsibley** [comment]
I have several more features that build off of this branch/PR, including:

* A Dockerfile for containerizing the web service
* Code 128 barcode support from the web form and API
* API documentation on the web page
* Optional gridline visibility
* An LCRY-2380 template

Would you like these on one big branch or separate?

**Comment by silby** [comment]
One big branch is fine; I have no particular interest in spending more time than strictly necessary with the GitHub pull request interface.

**Comment by silby** [comment]
I merged this one though.

**Comment by tsibley** [comment]
Ok. I'll push up a branch and make a single PR. You can always pull the branch from my fork and just work directly with it instead of the PR.

**Pull request closed without merging**